@sbidoul, I'm afraid they are related:

```
account_asset_management/tests/test_account_asset_management.py:143:80: E501 line too long (80 > 79 characters)
account_asset_management/tests/test_account_asset_management.py:146:80: E501 line too long (80 > 79 characters)
account_asset_management/tests/test_account_asset_management.py:159:80: E501 line too long (80 > 79 characters)
account_asset_management/tests/test_account_asset_management.py:162:80: E501 line too long (80 > 79 characters)
account_asset_management/tests/test_account_asset_management.py:233:80: E501 line too long (80 > 79 characters)
account_asset_management/tests/test_account_asset_management.py:236:80: E501 line too long (80 > 79 characters)
account_asset_management/tests/test_account_asset_management.py:241:80: E501 line too long (80 > 79 characters)
account_asset_management/tests/test_account_asset_management.py:244:80: E501 line too long (80 > 79 characters)
```

These are the real errors. The rest are only warnings. And they're introduced by your changes. You are using 80 chars instead of 79. The limitation is 80 cols, but this means 79 chars.

--- a/account_asset_management/tests/test_account_asset_management.py
+++ b/account_asset_management/tests/test_account_asset_management.py
@@ -140,27 +140,27 @@ class TestAssetManagement(common.TransactionCase):
             self.cr, self.uid, [asset.id])
         asset.refresh()
         if calendar.isleap(date.today().year):
-            self.assertAlmostEqual(asset.depreciation_line_ids[1].amount, 46.44,
-                                   places=2)
+            self.assertAlmostEqual(asset.depreciation_line_ids[1].amount,
+                                   46.44, places=2)
         else:
-            self.assertAlmostEqual(asset.depreciation_line_ids[1].amount, 47.33,
-                                   places=2)
-        self.assertAlmostEqual(asset.depreciation_line_ids[2].amount, 55.55,
-                               places=2)
-        self.assertAlmostEqual(asset.depreciation_line_ids[3].amount, 55.55,
-                               places=2)
-        self.assertAlmostEqual(asset.depreciation_line_ids[4].amount, 55.55,
-                               places=2)
-        self.assertAlmostEqual(asset.depreciation_line_ids[5].amount, 55.55,
-                               places=2)
-        self.assertAlmostEqual(asset.depreciation_line_ids[6].amount, 55.55,
-                               places=2)
+            self.assertAlmostEqual(asset.depreciation_line_ids[1].amount,
+                                   47.33, places=2)
+        self.assertAlmostEqual(asset.depreciation_line_ids[2].amount,
+                               55.55, places=2)
+        self.assertAlmostEqual(asset.depreciation_line_ids[3].amount,
+                               55.55, places=2)
+        self.assertAlmostEqual(asset.depreciation_line_ids[4].amount,
+                               55.55, places=2)
+        self.assertAlmostEqual(asset.depreciation_line_ids[5].amount,
+                               55.55, places=2)
+        self.assertAlmostEqual(asset.depreciation_line_ids[6].amount,
+                               55.55, places=2)
         if calendar.isleap(date.today().year):
-            self.assertAlmostEqual(asset.depreciation_line_ids[-1].amount, 9.11,
-                                   places=2)
+            self.assertAlmostEqual(asset.depreciation_line_ids[-1].amount,
+                                   9.11, places=2)
         else:
-            self.assertAlmostEqual(asset.depreciation_line_ids[-1].amount, 8.22,
-                                   places=2)
+            self.assertAlmostEqual(asset.depreciation_line_ids[-1].amount,
+                                   8.22, places=2)
 
     def test_3_proprata_init_prev_year(self):
         """Prorata temporis depreciation with init value in prev year."""
@@ -230,16 +230,16 @@ class TestAssetManagement(common.TransactionCase):
                                places=2)
         # I check computed values in the depreciation board
         if calendar.isleap(date.today().year):
-            self.assertAlmostEqual(asset.depreciation_line_ids[2].amount, 44.75,
-                                   places=2)
+            self.assertAlmostEqual(asset.depreciation_line_ids[2].amount,
+                                   44.75, places=2)
         else:
-            self.assertAlmostEqual(asset.depreciation_line_ids[2].amount, 45.64,
-                                   places=2)
-        self.assertAlmostEqual(asset.depreciation_line_ids[3].amount, 55.55,
-                               places=2)
+            self.assertAlmostEqual(asset.depreciation_line_ids[2].amount,
+                                   45.64, places=2)
+        self.assertAlmostEqual(asset.depreciation_line_ids[3].amount,
+                               55.55, places=2)
         if calendar.isleap(date.today().year):
-            self.assertAlmostEqual(asset.depreciation_line_ids[-1].amount, 9.11,
-                                   places=2)
+            self.assertAlmostEqual(asset.depreciation_line_ids[-1].amount,
+                                   9.11, places=2)
         else:
-            self.assertAlmostEqual(asset.depreciation_line_ids[-1].amount, 8.22,
-                                   places=2)
+            self.assertAlmostEqual(asset.depreciation_line_ids[-1].amount,
+                                   8.22, places=2)

